backend/src/apiserver/client_manager/client_manager.go

```diff
@@ -35,25 +37,33 @@ import (
 )
 
 const (
-	minioServiceHost       = "MINIO_SERVICE_SERVICE_HOST"
-	minioServicePort       = "MINIO_SERVICE_SERVICE_PORT"
-	minioServiceRegion     = "MINIO_SERVICE_REGION"
-	minioServiceSecure     = "MINIO_SERVICE_SECURE"
-	pipelineBucketName     = "MINIO_PIPELINE_BUCKET_NAME"
-	pipelinePath           = "MINIO_PIPELINE_PATH"
-	mysqlServiceHost       = "DBConfig.Host"
-	mysqlServicePort       = "DBConfig.Port"
-	mysqlUser              = "DBConfig.User"
-	mysqlPassword          = "DBConfig.Password"
-	mysqlDBName            = "DBConfig.DBName"
-	mysqlGroupConcatMaxLen = "DBConfig.GroupConcatMaxLen"
-	mysqlExtraParams       = "DBConfig.ExtraParams"
-	archiveLogFileName     = "ARCHIVE_CONFIG_LOG_FILE_NAME"
-	archiveLogPathPrefix   = "ARCHIVE_CONFIG_LOG_PATH_PREFIX"
-	dbConMaxLifeTime       = "DBConfig.ConMaxLifeTime"
-
-	visualizationServiceHost = "ML_PIPELINE_VISUALIZATIONSERVER_SERVICE_HOST"
-	visualizationServicePort = "ML_PIPELINE_VISUALIZATIONSERVER_SERVICE_PORT"
+	minioServiceHost   = "MINIO_SERVICE_SERVICE_HOST"
+	minioServicePort   = "MINIO_SERVICE_SERVICE_PORT"
+	minioServiceRegion = "MINIO_SERVICE_REGION"
+	minioServiceSecure = "MINIO_SERVICE_SECURE"
+	pipelineBucketName = "MINIO_PIPELINE_BUCKET_NAME"
+	pipelinePath       = "MINIO_PIPELINE_PATH"
+
+	mysqlServiceHost       = "DBConfig.MySQLConfig.Host"
+	mysqlServicePort       = "DBConfig.MySQLConfig.Port"
+	mysqlUser              = "DBConfig.MySQLConfig.User"
+	mysqlPassword          = "DBConfig.MySQLConfig.Password"
+	mysqlDBName            = "DBConfig.MySQLConfig.DBName"
+	mysqlGroupConcatMaxLen = "DBConfig.MySQLConfig.GroupConcatMaxLen"
+	mysqlExtraParams       = "DBConfig.MySQLConfig.ExtraParams"
```

Comment:
It might cause a breaking change to our users for moving the db config to MySQLConfig. @chensun is it okay to make this change?

Reply:
Seems to be fine, as I don't think `backend/src/apiserver/config/config.json` is user-facing?

Reply:
Sounds good, thanks for confirming! We are still keeping the user-facing flag unchanged: https://github.com/kubeflow/pipelines/blob/master/backend/src/cache/client_manager.go#L112-L115